docker/Dockerfile

```diff
@@ -20,17 +20,19 @@ FROM docker.io/python:${PYTHON_VERSION}-slim as builder
 
 # install the OS build deps
 RUN apt-get update && apt-get install -y \
-    build-essential \
-    libffi-dev \
-    libjpeg-dev \
-    libpq-dev \
-    libssl-dev \
-    libwebp-dev \
-    libxml++2.6-dev \
-    libxslt1-dev \
-    rustc \
-    zlib1g-dev \
- && rm -rf /var/lib/apt/lists/*
+        build-essential \
+        libffi-dev \
+        libjpeg-dev \
+        libpq-dev \
+        libssl \
+        libssl-dev \
+        libwebp-dev \
+        libxml++2.6-dev \
+        libxslt1-dev \
+        openssl \
```

Comment:
is `openssl` actually a build dep? I'm not sure we want to start throwing random things in here just in case they get used.

Reply:
How do we figure this out?

Reply:
Given that we don't know if anything depends on `openssl`, even if we don't *think* so, I'm inclined to install it anyway for paranoia reasons since its a security thing

Reply:
Looks like openssl is already installed in the base image

Reply:
Yep! Already installed, but not necessarily updated since the base images only get updated ~monthly at best. So good for us to do it here.